src/modules/infobox/components/FeatureList.vue

```diff
@@ -26,12 +28,26 @@ const layerFeatureCategories = ref([])
 
 const i18n = useI18n()
 const store = useStore()
+const activeLayers = computed(() => store.state.layers.activeLayers)
 const isCurrentlyDrawing = computed(() => store.state.drawing.drawingOverlay.show)
 const selectedEditableFeatures = computed(() => store.state.features.selectedEditableFeatures)
 const selectedFeaturesByLayerId = computed(() => store.state.features.selectedFeaturesByLayerId)
+const lastClick = computed(() => store.state.map.clickInfo)
+
+// flag telling if more features could be loaded for a given layer ID
+const canLoadMore = computed(() => (layerId) => {
+    // if the app was loaded with pre-selected features there won't be a clickInfo to latch upon,
+    // so we won't be able to load more features where the user has previously selected these features (we don't know where it was)
+    return (
+        lastClick.value &&
+        selectedFeaturesByLayerId.value.find(
+            (featuresForLayer) => featuresForLayer.layerId === layerId
+        )?.featureCountForMoreData > 0
+    )
+})
```

Comment:
This would make more sense to move this to `FeatureListCategory.vue` with the `loadMoreResultForLayer()` function below. This would avoid to do the the search in active layer twice

Reply:
I placed it here because in the Category component we already lost track of which kinds of "item/feature" this is, meaning there could be layer features (those where we want to load more) and vector feature (GeoJSON, KML, etc... for which it isn't possible to load more)

Reply:
We could have the `canLoadMore()` here, as it makes sense to have it where the informations are present. Then, we could call `loadMoreResultForLayer()` in the category, since this is where the button is. This would avoid defining an emitter and search the active layers twice.